frontends/web/src/api/exchanges.ts

```diff
@@ -75,6 +75,15 @@ export const signAddress = (format: string, msg: string, accountCode: AccountCod
   return apiPost('exchange/pocket/sign-address', { format, msg, accountCode });
 };
 
+export type AddressVerificationResponse = {
+  status: 'ok' | 'error' | 'used-address';
+  error: string;
```

Comment:
I am not 100% sure isn't error optional?
I believe this part https://github.com/digitalbitbox/bitbox-wallet-app/pull/1908/files#diff-b9e42f5f6cd5c1b3463424f0f375839cb238781d1c57495b03ae5e7af983e012R1169